### PR TITLE
Add Livepatch in Landscape section on livepatch page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Updated some copy and added "Kernel livepatching in Landscape" section.

## QA
- Pull code and run `./run`
- Go to http://localhost:8001/server/livepatch
- Check the copy matches the copy doc

Copy doc: https://docs.google.com/document/d/1ObUSvkDcWOe01pmhh_Aryqxs87D8X2khbJbn3RgqZE0/edit#

### CODE REVIEW

**Comment by anthonydillon** [comment]
[Update /server/livepatch for landscape-self-service](https://trello.com/c/2qR9UYCA/2983-update-server-livepatch-for-landscape-self-service)

**Comment by grahambancroft** [comment]
Code and QA lgtm 👍

File: templates/server/livepatch.html [hunk 1]
```diff
@@ -76,17 +76,15 @@
     </div>
 </div>
 
-<section class="row strip-light">
+<section class="row strip-light row-kernal-livepatching">
```
Comment (grahambancroft) [inline templates/server/livepatch.html]: Should this be 'row—kernel-livepatching' ?

Comment (anthonydillon) [inline templates/server/livepatch.html]: Goos spot, done.

**Pull request merged**